Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2018,13 +2018,13 @@ extension BrowserViewController {
         // Condition: Update from leanplum server
         // Get the A/B test variant from leanplum server
         // and update onboarding user reasearch
-        LeanPlumClient.shared.onStartLPVariable = {(lpVariable) -> () in
-            let lpVariableValue = "\(String(describing: lpVariable?.boolValue()))"
-            Sentry.shared.send(message: "Onboarding Research: onStartLPVariable - LP State - \(LeanPlumClient.shared.lpState.rawValue) | Condition: Received update from LP server with variable value as - \(lpVariableValue)", tag: .leanplum, severity: .debug, description: "Condition: Received update from LP server with variable value as - \(lpVariableValue)")
-            LeanPlumClient.shared.onStartLPVariable = nil
-            print("lp Variable from server \(lpVariableValue)")
+        LeanPlumClient.shared.finishedStartingLeanplum = {
+            let lpVariableValue = LPVariables.showOnboardingScreenAB?.boolValue()
+            Sentry.shared.send(message: "Onboarding Research: onStartLPVariable - LP State - \(LeanPlumClient.shared.lpState.rawValue) | Condition: Received update from LP server with variable value as - \(String(describing: lpVariableValue))", tag: .leanplum, severity: .debug, description: "Condition: Received update from LP server with variable value as - \(String(describing: lpVariableValue))")
+            LeanPlumClient.shared.finishedStartingLeanplum = nil
+            print("lp Variable from server \(String(describing: lpVariableValue))")
             self.onboardingUserResearch?.updateTelemetry()
-            let screenType: OnboardingScreenType = !(lpVariable?.boolValue() ?? false) ? .versionV2 : .versionV1
+            let screenType: OnboardingScreenType = !(lpVariableValue ?? false) ? .versionV2 : .versionV1
```

Comment:
nit: do `(showScreenB ?? true) ? .versionV2 : versionV1` , although I am surprised the default is v2

Reply:
lpVariableValue has three values nil, true, false
nil -> show the default .versionV1 as we didn't get anything from server 
true -> show the .versionV1 
false -> show the .versionv2

Simplified logic
```
if lpVariableValue == nil {
   .versionV1
} else if lpVariableValue == true {
   .versionV1
} else if lpVariableValue == false {
   .versionV2
}
```

> although I am surprised the default is v2

Reason being I was focused on showing .versionV2 😅 and because false == showingScreenB or .version2 
I'd suggest naming it to showScreenA as thats what our control variable then final code will be something like. 

`let screenType: OnboardingScreenType = (showScreenA ?? true) ? .versionV1 : .versionV2`

If we change to showScreenB and value we get from LP server is true
then _(showScreenB ?? true) ? .versionV2 : versionV1_ will actually return .versionV2 where as because server is sending true it should return versionV1